src/pages/Transaction.vue

```diff
@@ -108,11 +108,23 @@ export default {
   computed: {
     ...mapGetters('delegates', ['delegates']),
     ...mapGetters('currency', { currencySymbol: 'symbol' }),
-    ...mapGetters('network', ['height'])
+    ...mapGetters('network', ['height']),
+
+    confirmations() {
+      return this.initialBlockHeight ? this.height - this.initialBlockHeight : this.transaction.confirmations
+    }
   },
 
-  async mounted() {
-    await this.prepareComponent()
+  watch: {
+    async currencySymbol() {
+      await this.updateAverage()
+    },
+
+    height() {
```

Comment:
Maybe `initialBlockWeight` could be refactored to a computed property.

Reply:
I don't see a way for that, since `initialBlockHeight` depends on the current network height but needs to stay static.

Reply:
Yes, I'm going to refactor this on a new PR.